pingouin/pairwise.py

```diff
@@ -414,10 +421,16 @@ def pairwise_tests(
             ef = compute_effsize(x=x, y=y, eftype=effsize, paired=paired)
 
             if return_desc:
-                stats.at[i, "mean(A)"] = np.nanmean(x)
-                stats.at[i, "mean(B)"] = np.nanmean(y)
-                stats.at[i, "std(A)"] = np.nanstd(x, ddof=1)
-                stats.at[i, "std(B)"] = np.nanstd(y, ddof=1)
+                if parametric:
+                    stats.at[i, "mean(A)"] = np.nanmean(x)
+                    stats.at[i, "mean(B)"] = np.nanmean(y)
+                    stats.at[i, "std(A)"] = np.nanstd(x, ddof=1)
+                    stats.at[i, "std(B)"] = np.nanstd(y, ddof=1)
+                else:
+                    stats.at[i, "median(A)"] = np.nanmedian(x)
+                    stats.at[i, "median(B)"] = np.nanmedian(y)
+                    stats.at[i, "IQR(A)"] = np.abs(np.diff(np.percentile(x, [75, 25]))).item()
```

Comment:
please use built-in scipy.stats.iqr function

Reply:
More useful thank you;

https://github.com/turkalpmd/pingouin_plot_shift/blob/a0b37e89efde94b132106ad474b7b9fa0c5ff248/pingouin/pairwise.py#L399

Reply:
I must learn displaying of changes 😄